pkg/agent/interfacestore/types.go

```diff
@@ -31,24 +31,32 @@ const (
 
 type InterfaceType uint8
 
+func (i InterfaceType) String() string {
```

Comment:
Why these changes are needed? I see `Response` has tags.

Reply:
Thanks for reminding. I used to use structs from this package but now change to another new one. Code deleted.